pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -110,7 +110,14 @@ type GitLabInterceptor struct {
 
 // CELInterceptor provides a webhook to intercept and pre-process events
 type CELInterceptor struct {
-	Filter string `json:"filter,omitempty"`
+	Filter   string       `json:"filter,omitempty"`
+	Overlays []CELOverlay `json:"overlays,omitempty"`
```

Comment:
I think we need to fix our validation below so that it accepts either filters or overalys. Right now I need to specify a `filter` even if only want to use `overlays`
https://github.com/tektoncd/triggers/blob/master/pkg/apis/triggers/v1alpha1/event_listener_validation.go#L134

Reply:
I hadn't thought about that use case before, and yes, it definitely sounds useful, I'll sort that out.

Reply:
Should be doable now, and I've updated the docs with an example and explanation.